.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea to improve flakeshot!
+title: ''
+labels: 'feature request'
+assignees: ''
+
+---
```

Comment:
is this needed? What is this feature called in markdown? I haven't seen this one before.

Reply:
outdated.